app/import.go

```diff
@@ -15,17 +15,20 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/mattermost/mattermost-server/v6/app/imports"
 	"github.com/mattermost/mattermost-server/v6/app/request"
 	"github.com/mattermost/mattermost-server/v6/model"
 	"github.com/mattermost/mattermost-server/v6/shared/mlog"
 )
 
+type ReactionImportData = imports.ReactionImportData // part of the app interface
```

Comment:
This PR is supposed to be a non-change, to prevent any problems, the only type that has been moved and is used by interfaces is alias'd here.